client/src/components/Dashboard.js

```diff
@@ -18,7 +18,7 @@ const Dashboard = () => {
 
   const handleStrike = () => {
     const nextValue = strikes + 1;
-    Boolean(nextValue === 3)
+    nextValue === 3
```

Comment:
this was just Jonathan being extra explicit I think, but since it should always return a T/F value it's fine cutting it out :+1: